docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
+diverse team of community members, documentarians, and core developers.
+
+Things the workgroup could do:
+
+- Develop the governance model for docs
+- Be the steering council for docs
+- Be the authors of doc guidelines
+- Be the authors/editors of docs
+- Request grant funding from PSF, if needed
+- Form an editorial board for docs including core devs, educators, and
+  documentarians in the Python community
+
+Active Time
+-----------
+
+This workgroup will be active indefinitely while supported by the Python
+Steering Council.
+
+Core Values & Internal Governance
+---------------------------------
+
+The core values for this group are:
+
+- Respect
+- Support
+- Positive Atmosphere
+- Constructiveness
+- Integrity
```

Comment:
Minor: does a working group need its own statement of values? I assume the values statement from the PSF would be adopted by default absent this section?

Reply:
I don't disagree that all PSF workgroups should share a common set of core values rather than every charter repeating them and/or inventing their own, but this is [required per the PSF charter template](https://wiki.python.org/psf/Example%20PSF%20Workgroup%20Charter) so again, I don't think its wise to change it ourselves without consulting the PSF.